app/controllers/users_controller.rb

```diff
@@ -1,4 +1,19 @@
 class UsersController < ApplicationController
   def index
+    @users = User.all
+  end
+
+  def create
+    @user = User.create(user_params)
+  end
+
+  def new
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/purvak-simform/DemoDangerTool/blob/650dc568f8b1a3f4f40cd63d63336677b4fbfbe5/app/controllers/users_controller.rb#L10"></span>Action <code>new</code> should appear before <code>create</code>.</td>
    </tr>
  </tbody>
</table>